tests/testdata/aws-vpc/go/main.go

```diff
@@ -1,7 +1,7 @@
 package main
 
 import (
-	"github.com/pulumi/pulumi-terraform-module/sdks/vpc"
+	"github.com/pulumi/pulumi-terraform-module/sdks/go/vpc/v5/vpc"
```

Comment:
This seems like a weird import path

Reply:
It does indeed, but I think this change was latent in the codebase for a while.

This is decided here:

https://github.com/pulumi/pulumi-terraform-module-provider/blob/84d34b87d4d5dee8d36244c814471fd601366473/pkg/modprovider/schema.go#L47

Correspondingly this has changed here:

https://github.com/pulumi/pulumi-terraform-module/pull/213

We can follow up to simplify it if desired.

Reply:
ah ok I think that makes sense then